static/swagger/eds-spec.json

```diff
@@ -0,0 +1,370 @@
+            "description": "Bad request.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "error": {
+                      "type": "string",
+                      "description": "Error message."
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "ws://api/es/v1/subscribe": {
+      "get": {
+        "summary": "Subscribe to Stream via Web Socket",
+        "description": "**This API should not be tried from the Swagger UI**.\nSubscribe to stream via web-socket based on stream, type, or query parameters. Provide either `stream` or `filters` but not both or neither. \nThis API keeps the connection live and continuously receives messages from the server based on the specified `stream` or `filters`.\n\"filters\" is a JSON format string with different fields. E.g :- <br>\nfilters={ <br>\n  \"action\":“add”, <br>\n  \"once\":”TRUE | FALSE”, <br>\n  \"queries\":[ <br>\n  “select * from XXXX where n1==v1”, <br>\n  “select * from MMMM where n2==v1 OR n3>5”, <br>\n  “select \\<path\\> from XXXX” <br>\n  ] <br>\n} <br>\nThe *path* is a dot separated fully qualified string representing a json path for an object. *once* flag (default is false) specifies if the filters needs to be applied just once. <br>\n*filters* can be dynamically added and removed by sending message data on web socket connection in JSON format. E.g :- <br>\n{\"action\": \"remove\", \"queries\": [ \"select * from XXXX where n1=v1\"]}\n",
+        "tags": [
+          "Event Subscription"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "stream",
+            "required": false,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The name of the stream to subscribe to."
+          },
+          {
+            "in": "query",
+            "name": "type",
+            "required": false,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The type of events to subscribe to."
+          },
+          {
+            "in": "query",
+            "name": "filters",
+            "required": false,
+            "schema": {
+              "type": "string"
+            },
+            "description": "An optional query parameter to filter events."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful response.",
+            "headers": {
+              "X-PhotonIQ-ES": {
+                "description": "Epoch time header.",
+                "schema": {
+                  "type": "string",
+                  "description": "Epoch time in seconds."
+                }
+              },
+              "X-PhotonIQ-CustomerId": {
+                "description": "Customer Id header.",
```

Comment:
Throughout, "ID," not "Id." Also, remove the double periods in throughout (see line 162).

Reply:
Done